web/client/src/library/components/input/Textfield.tsx

```diff
@@ -0,0 +1,49 @@
+import clsx from 'clsx'
+import React from 'react'
+import { isFalse, isTrue } from '~/utils'
+
+export interface PropsTextfield {
+  value?: string | number
+  type?: string
+  placeholder?: string
+  disabled?: boolean
+  autoFocus?: boolean
+  className?: string
+  onInput?: (e: React.ChangeEvent<HTMLInputElement>) => void
+  onKeyDown?: (e: React.KeyboardEvent<HTMLInputElement>) => void
+}
+
+export default React.forwardRef<HTMLInputElement, PropsTextfield>(
+  function Input(
+    {
+      type = 'text',
+      value = '',
+      placeholder,
+      className,
+      disabled = false,
+      autoFocus = false,
+      onInput,
+      onKeyDown,
+    }: PropsTextfield,
+    ref?: React.Ref<HTMLInputElement>,
+  ): JSX.Element {
+    return (
+      <input
+        ref={ref}
+        className={clsx(
+          'placeholder:text-neutral-300 dark:placeholder:text-neutral-700',
+          isTrue(disabled) && 'opacity-50 cursor-not-allowed',
```

Comment:
Is `isTrue()` needed here? It's not present at line 184 in Selector.tsx: https://github.com/TobikoData/sqlmesh/pull/1151/files#diff-0c4605cd8c3c53a54b557d9f959536740216efb4fd2a3ecbe2445d597e41be0dR184

Reply:
yes, it is not needed